### PR TITLE
store: deal with 404 froms the SSO store properly

### PR DESCRIPTION
**Pull request by mvo5**

### CODE REVIEW

File: store/userinfo.go [hunk 1]
```diff
@@ -50,6 +50,16 @@ func UserInfo(email string) (userinfo *User, err error) {
...
 	defer resp.Body.Close()
 
+	switch resp.StatusCode {
+	case 200:
+		// good
+		break
+	case 404:
+		return nil, fmt.Errorf("cannot find user %q", email)
+	default:
+		return nil, fmt.Errorf("unexpcted status code %d for %q", resp.StatusCode, email)
```
Comment (chipaca) [inline store/userinfo.go]: maybe use `respToError` here

Comment (pedronis) [inline store/userinfo.go]: also typo

**Comment by chipaca** [comment]
All the :+1:

**Comment by chipaca** [comment]
retest this please

**Pull request merged**